### DISCUSSION TITLE
启用插件后，正向代理失效

### DISCUSSION THREAD

**Discussion by wind0ta** [shadowsocks/shadowsocks-windows#3196] [Q&A] [answered]
局域网环境，需要正向代理才能上网（外网机器用的CCproxy）。现在开启了v2ray-plugin插件后，正向代理失效了，无法上网。

**Comment by chenshaoju** [comment, marked as answer]
你好，正向代理无法支持插件，因为没有这方面的接口。

而v2ray-plugin目前不支持前置代理。
